9c-internal/multiplanetary/network/thor.yaml

```diff
@@ -0,0 +1,344 @@
+    keyId: "7b912d9b-b682-4403-a794-2d6421d108c9"
+    publicKey: "04ab9e31a20d8dbf5042bfc26ce9d9ed9a0e32ad787a1e5aa3ae8188fa5143861535acc7132cd8e74d4c1f0b94f843575e3add6988d3ccb1f54d7c59fb9535d789"
+
+  txpool:
+    type: "local"
+
+  notification:
+    slack:
+      bot:
+        username: "Relay Bridge Test (Odin ↔ Thor)"
+      channel: "9c-relay-bridge-bot-test-rdb"
+
+  db:
+    enabled: true
+    size: "5Gi"
+
+  rdb:
+    enabled: true
+    defaultStartBlockIndex:
+      upstream: "11902092"
+      downstream: "3415778"
+
+bridgeServiceApi:
+  enabled: false
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: thor-spot_2c
+
+dataProvider:
+  enabled: false
+  rwMode: false
+  render: true
+  migrateDatabase: false
+
+  storage:
+    data: 50Gi
+    log: 1Gi
+
+  db:
+    local: true
+    database: ""
+    connectionString: ""
+    host: ""
+    port: 3306
+    user: user
+    password: ""
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: thor-spot_2c
+
+marketService:
+  enabled: false
+
+  db:
+    local: true
+    size: 1Gi
+
+  nodeSelector:
+    read:
+      eks.amazonaws.com/nodegroup: thor-spot_2c
+
+  env:
+  - name: DOTNET_gcServer
+    value: "1"
+  - name: WorkerConfig__SyncShop
+    value: "true"
+  - name: WorkerConfig__SyncProduct
+    value: "true"
+
+patrolRewardService:
+  enabled: false
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: thor-spot_2c
+
+  db:
+    local: true
+
+remoteHeadless:
+  count: 0
+
+  resources:
+    requests:
+      cpu: 1
+      memory: 12Gi
+
+  storage:
+    data: 50Gi
+
+  hosts:
+  - "thor-internal-rpc-1.nine-chronicles.com"
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: thor-spot_2c
+
+  loggingEnabled: true
+
+  extraArgs:
+  - --planet=thorInternal
+  - --remote-key-value-service
+  - --arena-participants-sync=false
+
+rudolfService:
+  enabled: false
+
+  config:
+    graphqlEndpoint: "http://thor-internal-rpc-1.nine-chronicles.com/graphql"
+    genesisBlockHash: "729fa26958648a35b53e8e3905d11ec53b1b4929bf5f499884aed7df616f5913"
+
+  db:
+    local: true
+
+  kms:
+    keyId: "3a301c8e-96c5-4b13-8dee-16575ba637c6"
+    publicKey: "04d237239d672698c89fc0788918b4c8610de8b65c5d02ebf3e1f31dcb891683410a97b51f59c05c41c141ad3a05324ba2c5afe32d5e4909e63fd4a700cd36cc53"
+
+  serviceAccount:
+    roleArn: "arn:aws:iam::319679068466:role/thor-internal-9c-rudolf-signer"
+
+  service:
+    enabled: false
+    securityGroupIds:
+    - "sg-0c865006315f5b9f0"
+    - "sg-0343e5c4514681670"
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: thor-spot_2c
+
+seed:
+  count: 1
+
+  hosts:
+  - "thor-internal-tcp-seed-1.nine-chronicles.com"
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: thor-spot_2c
+
+snapshot:
+  partition:
+    enabled: false
+    suspend: true
+  slackChannel: "bot-test"
+
+  image: "planetariumhq/ninechronicles-snapshot:git-45205b5ed6d978bb0dda6ad0b84fb0a393015711"
+
+  path: internal/thor
+
+  resources:
+    requests:
+      cpu: 1300m
+      memory: 10Gi
```

Comment:
can we save resources for seed?

Reply:
```
resources:
    requests:
      cpu: 50m
      memory: 300Mi
```

This should be enough based on the internal heimdall seed usage. PTAL!